### PR TITLE
multiple identity support

### PR DESCRIPTION
**Pull request by ekoby**
new `ziti-edge-tunnel` options
`-i|--identity` can be repeated multiple times to load multiple identities
`-I|--identity-dir` can be used to load all identities from a directory

### CODE REVIEW

File: programs/ziti-edge-tunnel/ziti-edge-tunnel.c [hunk 1]
```diff
@@ -15,15 +15,27 @@
...
 
 const char *cfg_types[] = { "ziti-tunneler-client.v1", "intercept.v1", "ziti-tunneler-server.v1", "host.v1", NULL };
-static ziti_options OPTS = {
-        .config_types = cfg_types,
-        .events = ZitiContextEvent|ZitiServiceEvent,
-        .event_cb = on_ziti_event,
-        .refresh_interval = 10, /* default refresh */
-};
+
+static long refresh_interval = 10;
```
Comment (dovholuknf) [inline programs/ziti-edge-tunnel/ziti-edge-tunnel.c:36]: 10 on purpose? shouldn't this be more like 300?

File: programs/ziti-edge-tunnel/ziti-edge-tunnel.c [hunk 2]
```diff
@@ -164,14 +197,21 @@ static int run_opts(int argc, char *argv[]) {
     while ((c = getopt_long(argc, argv, "i:v:r:d:n:",
                             run_options, &option_index)) != -1) {
         switch (c) {
-            case 'i':
-                OPTS.config = strdup(optarg);
+            case 'i': {
```
Comment (smilindave26) [inline programs/ziti-edge-tunnel/ziti-edge-tunnel.c:200]: what do you think about supporting a directory (and attempt to load all of the .json identity files in the dir).
Or a single file that contains a list of identity files (and other config info).
possible to add an identity without restarting? eventual IPC support, I guess...

Comment (dovholuknf) [reply programs/ziti-edge-tunnel/ziti-edge-tunnel.c:200]: possibly even having it be `~/.ziti/identities` might be nice. the good thing is that it makes it easy to move your identity 
 for test reasons... which is the bad thing too possibly.  ZDEW works like this and scans the folder for 'new' identities when it starts up

Comment (ekoby) [reply programs/ziti-edge-tunnel/ziti-edge-tunnel.c:200]: directory option is  coming

File: programs/ziti-edge-tunnel/ziti-edge-tunnel.c (hunk 1 above)
Comment (ekoby) [reply programs/ziti-edge-tunnel/ziti-edge-tunnel.c:36]: this was the default, I am not changing that

**Review by dovholuknf — APPROVED** [review]

**Pull request merged**